### PR TITLE
Update vague description of autoconfigure #515

### PR DESCRIPTION
**Pull request by Nek-**
| Q             | A
| ------------- | ---
| License       | MIT

See #515 for details.

### CODE REVIEW

File: symfony/framework-bundle/4.2/config/services.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
     # default configuration for services in *this* file
     _defaults:
         autowire: true      # Automatically injects dependencies in your services.
-        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+        autoconfigure: true # Automatically adds corresponding tags to your services depending on interfaces they implement.
```
[the remark this answers is not in the document]
Comment (B-Galati) [reply symfony/framework-bundle/4.2/config/services.yaml]: I think we could add a parenthesis like so:

```
Automatically registers your services as commands, event subscribers, etc. (By adding tags, method calls, etc.) when class type is supported.
```

[the remark this answers is not in the document]
Comment (Pierstoval) [reply symfony/framework-bundle/4.2/config/services.yaml]: What about `Automatically adds useful options to services that extend certain classes` and add a link to https://symfony.com/blog/new-in-symfony-3-3-service-autoconfiguration ?

[the remark this answers is not in the document]
Comment (OskarStark) [reply symfony/framework-bundle/4.2/config/services.yaml]: @B-Galati I like your proposal 👍🏻

**Comment by B-Galati** [comment]
@Nek- Given what @nicolas-grekas said I would say no.

**Comment by nicolas-grekas** [comment]
What do you think of:
`Automatically tag services, call their setters, etc. depending on the types they implement.`

**Comment by B-Galati** [comment]
@nicolas-grekas IMHO, the actual version is clearer because it focused on behavior while the one you proposed focused on implementation details, which is great but not the main point, IMHO again.

Another proposal:

>Automatically registers your services as commands, event subscribers, etc. (By tagging services, calling their setters, etc.) depending on the types they implement.

**Comment by Nek-** [comment]
Closing because this kind of things should not take that much of our time.

**Pull request closed without merging**